packages/babel-plugin-named-asset-import/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "babel-plugin-named-asset-import",
+  "version": "0.1.0",
+  "description": "Babel plugin for named asset imports in Create React App",
+  "repository": "facebookincubator/create-react-app",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/facebookincubator/create-react-app/issues"
+  },
+  "main": "index.js",
+  "files": [
+    "index.js"
+  ],
+  "dependencies": {
+    "@babel/core": "7.0.0-beta.38"
```

Comment:
Why? Maybe this should be a peer dependency?

Reply:
I originally wrote this plugin outside create-react-app so I needed this. I can change it to a peer dependency.